frontend/src/components/main/main.js

```diff
@@ -1,32 +1,34 @@
 import React, { useEffect, useState } from 'react';
 import './main.css';
 import axios from 'axios';
+import Slider from './Slider/Slider';
+import Game from '../game/game'
 
 function MainPage() {
+    const email = sessionStorage.getItem("email");
+    const pos = email.indexOf('@');
+    const user = email.substr(0,pos);
    
     return(
-        <div>
-            {(sessionStorage.getItem("login")) ? <h1>YOU ARE LOGGED IN</h1> : <h1>YOU ARE NOT LOGGED IN</h1>}
+        <div className="mainDiv">
+            {(sessionStorage.getItem("login")) ?
+                <div class="successMain">
+                    <span><h1>Welcome back </h1><h1 id="userWelcome">{user}!</h1></span>
+                    <Slider />  
+                    <div className="recentGames">
+                        <h1 className="recentGamesHeader">Recent Games: </h1>   
+                        <Game />
+                        <Game />  
+                        <Game />  
+                        <Game />   
+                    </div>       
+                </div>
+                
+                
+                : <h1>YOU ARE NOT LOGGED IN</h1>}
+            
```

Comment:
Nice, but it looks much more impressive now!

Reply:
Thanks! We'll get there 👍

Reply:
I meant, 'bet' not 'but'.